### PR TITLE
Update dropbox.md

### PR DESCRIPTION
**Pull request by jerry-li-dev**

### CODE REVIEW

File: src/pages/docs/integrations/available-integrations/dropbox.md [hunk 1]
```diff
@@ -17,30 +17,52 @@ Backup and synchronize your Postman Collections on Dropbox for file sharing, sto
...
+1. Select **Add Integration**. Your collection is saved as a single JSON file and uploaded to Dropbox.
 
 ## Accessing your Postman Collections on Dropbox
 
 You can find your backups on Dropbox > Apps > Postman.
 
-[![view in dropbox](https://assets.postman.com/postman-docs/dropbox_view.png)](https://assets.postman.com/postman-docs/dropbox_view.png)
+  ![view in dropbox](https://assets.postman.com/postman-docs/dropbox_view.jpg)
 
 From now on, every change saved to your Postman Collection will automatically upload changes to Dropbox in real time. Complement your team's existing workflows and backup your Postman Collections to Dropbox.
```
Comment (jkonrath-postman) [inline src/pages/docs/integrations/available-integrations/dropbox.md:68]: backup/back up

File: src/pages/docs/integrations/available-integrations/dropbox.md [hunk 2]
```diff
@@ -17,30 +17,52 @@ Backup and synchronize your Postman Collections on Dropbox for file sharing, sto
...
-   Your collection is saved as a single JSON file and uploaded to Dropbox.
+    ![dropbox add integration](https://assets.postman.com/postman-docs/dropbox-add-integration.jpg)
+
+1. Select **Continue**.
+
+    ![authorize to go to dropbox login page](https://assets.postman.com/postman-docs/dropbox-warning.jpg)
+
+1. Log in to your Dropbox account.
+
+    ![dropbox login](https://assets.postman.com/postman-docs/dropbox-login.jpg)
```
Comment (jkonrath-postman) [inline src/pages/docs/integrations/available-integrations/dropbox.md:46]: Old screenshot - still mentions Postman Pro

File: src/pages/docs/integrations/available-integrations/dropbox.md [hunk 3]
```diff
@@ -17,30 +17,52 @@ Backup and synchronize your Postman Collections on Dropbox for file sharing, sto
...
+
+1. Select **Allow** so Postman can access its own folder inside your Dropbox.
+
+    ![allow postman access](https://assets.postman.com/postman-docs/dropbox-allow.jpg)
```
Comment (jkonrath-postman) [inline src/pages/docs/integrations/available-integrations/dropbox.md:50]: Also old screenshot

File: src/pages/docs/integrations/available-integrations/dropbox.md [hunk 4]
```diff
@@ -17,30 +17,52 @@ Backup and synchronize your Postman Collections on Dropbox for file sharing, sto
...
+    ![dropbox integration config](https://assets.postman.com/postman-docs/dropbox-save-config.jpg)
+
+1. Select **Add Integration**. Your collection is saved as a single JSON file and uploaded to Dropbox.
 
 ## Accessing your Postman Collections on Dropbox
 
 You can find your backups on Dropbox > Apps > Postman.
 
-[![view in dropbox](https://assets.postman.com/postman-docs/dropbox_view.png)](https://assets.postman.com/postman-docs/dropbox_view.png)
+  ![view in dropbox](https://assets.postman.com/postman-docs/dropbox_view.jpg)
```
Comment (jkonrath-postman) [inline src/pages/docs/integrations/available-integrations/dropbox.md:66]: Old screenshot. (You might be able to just edit that by hand to remove the "Pro" from the Postman Pro folder.)

**Review by jkonrath-postman — CHANGES_REQUESTED** [review]
A couple of little changes, but otherwise good

File: src/pages/docs/integrations/available-integrations/dropbox.md (hunk 2 above)
Comment (jerry-li-dev) [reply src/pages/docs/integrations/available-integrations/dropbox.md:46]: This is a new screenshot I created. Dropbox still calls it Postman Pro.

File: src/pages/docs/integrations/available-integrations/dropbox.md (hunk 3 above)
Comment (jerry-li-dev) [reply src/pages/docs/integrations/available-integrations/dropbox.md:50]: Same thing here. Dropbox still calls it Postman Pro.

File: src/pages/docs/integrations/available-integrations/dropbox.md (hunk 4 above)
Comment (jerry-li-dev) [reply src/pages/docs/integrations/available-integrations/dropbox.md:66]: Same thing here. Dropbox still calls it Postman Pro.

**Review by jkonrath-postman — APPROVED** [review]
Looks good to me

**Pull request merged**